docs/sources/clients/promtail/scraping.md

```diff
@@ -4,6 +4,17 @@ description: Promtail Scraping (Service Discovery)
 ---
 # Scraping
 
+Promtail currently supports scraping from the following sources:
+- Files
```

Comment:
Why don't we go ahead and make each bullet point a hyperlinks and save some more scrolling?
```suggestion
- {{< relref "#file-target-discovery" >}}
```